### ISSUE TITLE
[Feature Request] Allow surahs to be saved offline

### ISSUE THREAD

**Issue by ammarbinfaisal** [quran/quran.com-frontend#1041]
I see that the website is using a service worker which does nothing and there are a few issues open to make this a PWA #975 
But i don't just mean that it should just be a pwa.
There should an option to save data offline like there's in [devdocs](https://devdocs.io)

I would like to help in making this a pwa in which surahs can be saved.

**Comment by mrtugicom** [comment]
Yeah cool, also to import data, so we dont need to download from server.
PWA with export import capabilities for sure?
Enhance portability across platforms and devices.
